### PR TITLE
Add Community translations section

### LINKED ISSUE #117: Feature: Add Community Translations Section
## Type of feature

🍕 Feature

## Current behavior

Although it's not possible for OpenSauced to create and maintain all requested translations, we value community translation contributions.

To recognize these contributions and let our users know that we are not specifically maintaining them, we want to create a space for those willing to maintain translations in any language outside OpenSauced official translations to link to their translations.

_original issue by @BekahHW: https://github.com/open-sauced/intro/issues/71#issue-1935780130_

### Create New File

- [ ] Create `community-translations.md` at the root of the repo. In that file, create the following:

   ```markdown
   # Community Course Translations

   We are grateful for the contributions of the community in translating this course. Although the OpenSauced team does not currently maintain these projects, we want to share the forked copy of the repositories with those taking the course. Below, you will find the latest community translations.
   ```

### Adjustment

With a new file added, some content need to be adjusted in two files:

#### 1. README

- [ ] Adjust the "Language Support" section in the README as follow:

   ```markdown
   ## Language Support

   Like pizza, open source is meant to be shared with everyone. We provide and maintain translations in languages below as our official translations:

   - [French](./translations/fr/README.md)

   For translations in other languages that are maintained by our community, please go to the [Community Translations](link to community-translations.md) section.
   ```

   **Code snippet:**
   https://github.com/open-sauced/intro/blob/59b55ca73787bbadda275d8b4caabd5e7be0595b/README.md?plain=1#L11-L16

#### 2. `i18n-guidelines.md`

- [ ] Change the word "OSS" to "Open Source".
- [ ] Add the link to the Community Translation section.

```diff
- We can't always support the maintenance of translations. However, we know some contributors are willing to translate, and we value these contributions. For that reason, we have a Community Translations section.
-
- If you're interested in translating our Intro to OSS course, fork this repository and add the translation to your forked repository. Then, you can add a link to your translation in the `community-translations.md` file.
-
+ We can't always support the maintenance of translations. However, we know some contributors are willing to translate, and we value these contributions. For that reason, we have a [Community Translations](link to the section) section.
+
+ If you're interested in translating our Intro to Open Source course, fork this repository and add the translation to your forked repository. Then, you can add a link to your translation in the `community-translations.md` file that you can find in the root directory.
+
```

**Code snippet:**
https://github.com/open-sauced/intro/blob/59b55ca73787bbadda275d8b4caabd5e7be0595b/i18n-guidelines.md?plain=1#L28-L30

## Additional context

_No response_

## Code of Conduct

- [X] I agree to follow this project's Code of Conduct

## Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by vivitt**
## Description

This PR creates a Community Translations section to provide a space where those willing to maintain translations in any language outside OpenSauced official translations can link to their translations.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Closes #117 

### Community Course Translations
Screenshot 2024-03-12 at 13 13 14

### Readme
Screenshot 2024-03-12 at 19 01 54

### i18-guidelines
Screenshot 2024-03-12 at 19 02 58

## Steps to QA

1. Navigate to Home Page and note the changes in the *Language Support* section.
2. Click the *Community Translations* link in the Language Support section to navigate to the new section.
3. Navigate to i18n Guidelines and note the changes in the *2. Community Translations* sections.
4. Click the *Community Translations* link in the Language Support section to navigate to the new section.

## Added to documentation?

- [x] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: community-translations.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Community Course Translations
+
+We are grateful for the contributions of the community in translating this course. Although the OpenSauced team does not currently maintain these projects, we want to share the forked copy of the repositories with those taking the course. Below, you will find the latest community translations.
```
Comment (adiati98) [inline community-translations.md:3]: Can you please add "Intro to Open Source" in the first sentence for clarity?

You can simply click the "Commit" button below this suggestion to accept and commit the changes. :)

```suggestion
We are grateful for the contributions of the community in translating this Intro to Open Source course. Although the OpenSauced team does not currently maintain these projects, we want to share the forked copy of the repositories with those taking the course. Below, you will find the latest community translations.
```

**Review by adiati98 — COMMENTED** [review]
@vivitt thank you so much for working on this! ✨
The PR looks great! I only have one review here for clarity purpose.

Once it's done, you can let me know by tagging me in the comment so I can approve your PR.
Thank you! 😄

**Comment by vivitt** [comment]
Done! Thank you @adiati98 for your help :)

**Review by BekahHW — APPROVED** [review]

**Review by adiati98 — APPROVED** [review]
Well done, @vivitt ! 🙌
Thank you for the changes. 😄

If you haven't, you can join our community [Discord](https://discord.com/invite/U2peSNf23P). 😊

Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕 

git merge GIF

**Pull request merged**